src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckSwitchExpressionsFour.java

```diff
@@ -0,0 +1,23 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+class InputMissingSwitchDefaultCheckSwitchExpressionsFour {
+    public void switchWithNullTest(SwitchInput i) {
+        switch (i) {
+            case FIRST -> System.out.println("FIRST");
+            case SECOND -> System.out.println("SECOND");
+            case null -> System.out.println("NULL");
```

Comment:
We need to extend test cases to have multiple case constants in one label, null case constants in switch expressions, null case constants in non-exhaustive switch statements.

Reply:
@nrmancuso More Test case added